rpc/api.go

```diff
@@ -135,6 +135,64 @@ func (sc *Client) GetStorageAt(ctx context.Context, contractAddress, key, blockH
 	return value, err
 }
 
+// StorageDiff is a change in a single storage item
+type StorageDiff struct {
+	// ContractAddress is the contract address for which the state changed
+	ContractAddress string `json:"address"`
```

Comment:
I think we should try align with rpc naming, even tho it is a bit inconsistent itself

```suggestion
	Address string `json:"address"`
```